src/hooks/search/index.ts

```diff
@@ -47,67 +51,93 @@ const useSearch = (
 
     // TODO: Refactor to useSWRInfinite and implement client-side pagination.
     const { data, error, isValidating } = useSWR(key, fetcher, {
-        revalidateIfStale: false,
+        // revalidateIfStale: false,
         revalidateOnFocus: false,
         revalidateOnReconnect: false,
         shouldRetryOnError: false,
+        fallback: {
```

Comment:
There previously was some verbose code related to setting initial data for SEO purposes, but I found this `fallback` option on the [Vercel docs](https://swr.vercel.app/docs/with-nextjs#pre-rendering-with-default-data) and it seems to have the same performance/SEO benefits while allowing for much cleaner code. This data seems to be statically generated in the build output so it shouldn't break SEO, but let me know if I'm missing anything.